libs/src/utils/network.ts

```diff
@@ -60,50 +76,62 @@ function sortServiceTimings ({
       if (semver.lt(aVersion, bVersion)) return 1
     } else {
       // Only sort by version if behind current on-chain version
+      // @ts-expect-error
```

Comment:
how come? maybe a comment to explain

Reply:
good call, will take another look, just added that early on and forgot